framework/src/multiapps/MultiApp.C

```diff
@@ -138,9 +138,16 @@ validParams<MultiApp>()
   params.addParam<std::vector<Point>>("move_positions",
                                       "The positions corresponding to each move_app.");
 
+  params.addParam<std::vector<std::string>>(
+      "cli_args",
```

Comment:
How does this interact and work with the existing way we pass arguments to the Multiapp?
https://github.com/idaholab/moose/tree/next/test/tests/parser
You should take a look at merging or making sure these two implementations play well together.

Reply:
This uses the existing system, by appending items to the command line argument when the sub-app is created within MultiApp.

Reply:
OK